chromium_src/third_party/blink/common/client_hints/enabled_client_hints.cc

```diff
@@ -15,7 +15,15 @@ namespace blink {
 
 void EnabledClientHints::SetIsEnabled(const WebClientHintsType type,
                                       const bool should_send) {
-  enabled_types_[static_cast<int>(type)] = false;
+  switch (type) {
+    case WebClientHintsType::kUA:
```

Comment:
What value do we send for this one? The Chromium one or the Chrome one?
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Sec-CH-UA#examples

Reply:
Ok, so we will look different from Chrome then.

Reply:
@mkarolin shouldn't this be the same as whats in navigator.userAgentData (which is different from both Chrome and Chromium)

Reply:
Doesn't seem like it. There's a whole bunch of code that determines branding for client hints, specifically in [blink::UserAgentBrandList GetUserAgentBrandList](https://source.chromium.org/chromium/chromium/src/+/main:components/embedder_support/user_agent_utils.cc;l=233). We get the Chromium one because we don't unset `CHROMIUM_BRANDING` when building. If we unset it we would get `".Not/A)Brand";v="99", "Brave Browser";v="103", "Chromium";v="103"`

Reply:
I think thats what we want, as long as it wouldn't change the standard UA string (do you know if it would?)

Reply:
Doesn't appear so, the UA string is `Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36`

Reply:
Added a patch to apply branding to the UA CH.